### PR TITLE
Migrate Images block

### PR DESCRIPTION
**Pull request by saugatmalla**
* Migrate Images block
* Refactored
* Added test

Resolves: Images block migration

**Test URLs:**
- Before:  https://main--milo--adobecom.hlx.page/drafts/saugat/figure?martech=off
- After: https://images--milo--adobecom.hlx.page/drafts/saugat/figure?martech=off

### CODE REVIEW

File: libs/blocks/images/images.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { createTag } from '../../utils/utils.js';
+
+/**
+* Initializes the image block by wrapping each picture element in a div with a class of "figure"
```
Comment (vhargrave) [inline libs/blocks/images/images.js:4]: nit: in my opinion we don't need this comment :)

File: libs/blocks/images/images.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+*@param {Element} blockEl - The block element to initialize.
+*/
+export default function init(blockEl) {
```
Comment (vhargrave) [inline libs/blocks/images/images.js:7]: can you run npm run lint on this file and fix the linting errors? 
On line 11 you're missing a semicolon after your return statement for example.

**Review by vhargrave — APPROVED** [review]

**Review by chrischrischris — APPROVED** [review]

**Review by auniverseaway — CHANGES_REQUESTED** [review]
We're missing the whole figure creation piece...

https://github.com/adobe/blog/blob/main/blocks/images/images.js#L18

**Comment by auniverseaway** [comment]
Alright. I spoke with the big guns, we should change this block name to `figure`.

**Comment by saugatmalla** [comment]
@auniverseaway I added `buildFigure` method and change the block name to `figure`.

@vhargrave @chrischrischris FYI

**Review by auniverseaway — APPROVED** [review]

**Pull request merged**